.travis.yml

```diff
@@ -3,7 +3,7 @@ language: ruby
 cache: bundler
 
 sudo: false
-
+before_script: "unset _JAVA_OPTIONS"
```

Comment:
leaked in?

Reply:
Yep, causes a failure https://github.com/travis-ci/travis-ci/issues/8408#issuecomment-344087255 and https://github.com/documentcloud/closure-compiler/issues/48